tests/model/test_instance.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
+import pytest
+from sleap_io.model.instance import (
+    Point,
+    PredictedPoint,
+    Track,
+    Instance,
+    LabeledFrame,
+    PredictedInstance,
+)
+from sleap_io.model.skeleton import Skeleton, Node
+from tests.fixture.fixtures import test_video
+
+
+def test_classes(test_video):
+
+    point = Point(x=0, y=0)
+    pred_point = PredictedPoint.from_point(point)
+    track = Track()
+    skeleton = Skeleton.from_names(
+        nodes=["head", "thorax", "abdomen"],
+        edges=[("head", "thorax"), ("thorax", "abdomen")],
+    )
+    instance1 = Instance(skeleton=skeleton, points={"head": point})
+    pointsarray1 = np.array([[1, 1], [2, 2], [3, 3]], dtype="float32")
+    pointsarray2 = np.array(
+        [[1, 1, True, False], [2, 2, True, False], [3, 3, True, False]], dtype="float32"
+    )
+    pointsconfidence = np.array([1, 2, 3], dtype="float32")
+    instance2 = Instance.from_pointsarray(points=pointsarray1, skeleton=skeleton)
+    instance3 = Instance.from_pointsarray(points=pointsarray2, skeleton=skeleton)
+
+    pred_instance1 = PredictedInstance(skeleton)
+    pred_instance2 = PredictedInstance.from_instance(instance1, 0.0)
+    pred_instance3 = PredictedInstance.from_arrays(
+        points=pointsarray1,
+        point_confidences=pointsconfidence,
+        instance_score=0.0,
+        skeleton=skeleton,
+    )
+    pred_instance4 = PredictedInstance.from_arrays(
+        points=[np.nan],
+        point_confidences=pointsconfidence,
+        instance_score=0.0,
+        skeleton=skeleton,
+    )
+    dummy = test_video
+    labeledframe_ = LabeledFrame(video=dummy, frame_idx=1, instances=[instance1])
```

Comment:
Use snake case as per PEP8. Nit pick: Also why the [trailing underscore](https://peps.python.org/pep-0008/#descriptive-naming-styles:~:text=single_trailing_underscore_%3A%20used%20by%20convention%20to%20avoid%20conflicts%20with%20Python%20keyword%2C%20e.g)?